tuf/repository_tool.py

```diff
@@ -3157,35 +3157,65 @@ def append_signature(signature, metadata_filepath):
   file_object.move(metadata_filepath)
 
 
-# Wrapper functions that we wish to make available here from repository_lib.py.
+# Wrapper functions that we wish to make available here from securesystemslib.
 # Users are expected to call functions provided by repository_tool.py.  We opt
-# for this approach, as opposed to using import statements to achieve the
+# for wrapper functions, instead of using the import statements to achieve the
 # equivalent, to avoid linter warnings for unused imports.
-def generate_and_write_rsa_keypair(filepath, bits, password):
-  return repo_lib.generate_and_write_rsa_keypair(filepath, bits, password)
-
 def generate_and_write_ed25519_keypair(filepath, password):
   return repo_lib.generate_and_write_ed25519_keypair(filepath, password)
 
-def import_rsa_publickey_from_file(filepath):
-  return repo_lib.import_rsa_publickey_from_file(filepath)
+def generate_ed25519_key():
+  return securesystemslib.keys.generate_ed25519_key()
 
 def import_ed25519_publickey_from_file(filepath):
   return repo_lib.import_ed25519_publickey_from_file(filepath)
 
+def import_ed25519_privatekey_from_file(filepath, password):
+  return repo_lib.import_ed25519_privatekey_from_file(filepath, password)
+
+# NOTE: securesystemslib cannot presently import an Ed25519 key from PEM.
+
+def generate_and_write_rsa_keypair(filepath, bits, password):
+  return repo_lib.generate_and_write_rsa_keypair(filepath, bits, password)
+
+def generate_rsa_key():
+  return securesystemslib.keys.generate_rsa_key()
+
+def import_rsa_publickey_from_file(filepath):
+  return repo_lib.import_rsa_publickey_from_file(filepath)
+
```

Comment:
Is there a reason why some of these calls are passed through an equivalent wrapper in `repository_lib`? I suggest you directly call `securesystemslib` (as you already do in some functions) and shorten `repository_lib`, which would make maintenance easier. Or does that change the API too much?